marshmallow_dataclass/__init__.py

```diff
@@ -84,29 +83,35 @@ class User:
 
 @overload
 def dataclass(
-    _cls: Type[_U],
-    *,
-    repr: bool = True,
-    eq: bool = True,
-    order: bool = False,
-    unsafe_hash: bool = False,
-    frozen: bool = False,
-    base_schema: Optional[Type[marshmallow.Schema]] = None,
-    cls_frame: Optional[types.FrameType] = None,
+        _cls: Type[_U],
```

Comment:
Why did the indentation change?

Reply:
I have the auto-indentation on and it automatically corrected the indentation on save.

Reply:
4 spaces is a lot more standard than 8. Since changing the indentation is not relevant to this PR I think that should be adjusted.

Reply:
See #205; adds version guard switching.

Reply:
@mivade fixed indentation, could you please check now?